yarn.lock

```diff
@@ -6122,7 +6103,7 @@ lodash._root@^3.0.0:
   resolved "https://registry.yarnpkg.com/lodash._root/-/lodash._root-3.0.1.tgz#fba1c4524c19ee9a5f8136b4609f017cf4ded692"
   integrity sha1-+6HEUkwZ7ppfgTa0YJ8BfPTe1pI=
 
-lodash.assign@4.2.0, lodash.assign@^4.2.0:
+lodash.assign@^4.2.0:
```

Comment:
I'm wondering why those are still here?

Reply:
`yarn why lodash.assign` tells us that it's a dependency of `_project_#webdriverio#archiver#archiver-utils`

Reply:
OK so just for tests. 
Sounds good to me.